src/components/other/TabularSheet/EditField/index.js

```diff
@@ -52,6 +76,7 @@ class EditFieldModal extends React.PureComponent {
             options: { $auto: {
                 separator: { $setDefault: 'none' },
                 geoType: { $setDefault: 'name' },
+                dateFormat: { $setDefault: undefined },
```

Comment:
Setting default to undefined is un-necessary

Reply:
fixed